plural/helm/plural/values.yaml.tpl

```diff
@@ -95,7 +90,9 @@ hydra:
     public:
       hosts:
       - host: {{ $hydraHost }}
-        paths: ["/.*"]
+        paths:
+        - path: /
```

Comment:
This probably needs to be `/.*` still if we've enabled regex for that ingress

Reply:
I think this is covered by `pathType: ImplementationSpecific`.